### ISSUE TITLE
encounting the below error, please help me

### ISSUE THREAD

**Issue by githublsk** [onnx/onnx#4330]
when i covert pytoch model to onnx, the error occur as below, I think because entities and relations is a dict format data, which I set them with wrong shape?
![image](https://user-images.githubusercontent.com/77612906/176648103-d0a64955-e04c-4cb6-bf46-e48dfc939fd2.png)
I set the input as follows:
![image](https://user-images.githubusercontent.com/77612906/176648250-9698de91-33df-4145-9677-219d57de9336.png)
the dummy input data is as follows:
![image](https://user-images.githubusercontent.com/77612906/176648417-177fe839-f846-4ae4-b93f-702ff4d3eccc.png)

**Comment by jcwchen** [comment]
Hi @githublsk,
According to [the exporter doc](https://pytorch.org/docs/stable/onnx.html#contributing-developing), perhaps you need to add an empty dictionary in the end. Still, this error happened while using `torch.onnx.export` so it should be the question to the PyTorch-ONNX exporter. If my suggestion does not help, please file this issue in their repo instead to get faster/better assistance from the exporter experts.

**Comment by wschin** [comment]
Dynamic axis means axis of a tensor. `eneities` and `relations` are not tensors so it may throw. If possible, I'd suggest to encode your data as tensors instead of dictionary; for example,
```
labels = torch.zero...
image= torch.zero...
starts= torch.tensor([1,3,4,5])...
ends=...
head=...
tail=...
```
